devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/buildtool/gradle-kotlin-dsl/codestart.yml

```diff
@@ -14,15 +14,16 @@ language:
           id: io.quarkus
       buildtool:
         build-dir: build
+        cli: gradle
         guide: https://quarkus.io/guides/gradle-tooling
         guide-native: https://quarkus.io/guides/gradle-tooling#building-a-native-executable
         cmd:
-          dev: gradle quarkusDev
-          package: gradle quarkusBuild
-          package-uberjar: gradle quarkusBuild --uber-jar
-          package-native: gradle build -Dquarkus.package.type=native
-          package-native-container: gradle build -Dquarkus.package.type=native -Dquarkus.native.container-build=true
-          build-ci: gradle build
+          dev: quarkusDev
+          package: quarkusBuild
+          package-uberjar: quarkusBuild --uber-jar
+          package-native: build -Dquarkus.package.type=native
+          package-native-container: build -Dquarkus.package.type=native -Dquarkus.native.container-build=true
+          build-ci-args: build
```

Comment:
@glefloch why keeping `args` for this one only?

Reply:
yes you are right, I removed it.